projects/imagemagick/project.yaml

```diff
@@ -0,0 +1,5 @@
+homepage: "https://www.imagemagick.org"
+primary_contact: "<primary_contact_email>"
```

Comment:
+1, would be great to have @dlemstra or another maintainer here

Reply:
You can use dirk at git.imagemagick.org for now.